create-ava-rule.js

```diff
@@ -129,16 +130,29 @@ module.exports = function createAvaRule() {
 		}
 	};
 
-	Object.defineProperty(rule, 'isTestFile', {
-		get: function () {
-			return isTestFile;
-		}
-	});
+	rule.isInTestFile = function () {
+		return isTestFile;
+	};
 
-	Object.defineProperty(rule, 'currentTestNode', {
-		get: function () {
-			return currentTestNode;
-		}
+	rule.isTestNode = function (node) {
+		return currentTestNode === node;
+	};
+
+	rule.isInTestNode = function () {
+		return currentTestNode;
+	};
+
+	rule.if = rest(function (predicates) {
```

Comment:
I prefer `if`. We don't care about ES3. If your editor is highlighting it incorrectly, I'd suggest opening and issue/PR on your editor.

Sublime is very open to pull requests for syntax highlighting now btw: https://github.com/sublimehq/Packages